### PR TITLE
update ppom: resolve #20

### LINKED ISSUE #20: Update to latest parent and asto
Update to the latest parent pom `0.3` and asto `0.14`, downgrade to JDK8.

### PR DESCRIPTION
**Pull request by Guseyn**

### CODE REVIEW

**Comment by g4s8** [comment]
@0crat assign me

File: src/main/java/com/artipie/npm/Npm.java [hunk 1]
```diff
@@ -161,4 +172,23 @@ private Completable updateMetaFile(
...
+     * @return Bytes from list of buffers.
+     */
+    private static byte[] bytesFromListOfByteBuffers(final List<ByteBuffer> buffers) {
+        final ByteArrayOutputStream output = new ByteArrayOutputStream();
+        buffers.forEach(
+            buffer -> {
+                try {
+                    output.write(new Remaining(buffer).bytes());
+                } catch (final IOException exp) {
+                    LOGGER.info(exp.getMessage());
```
Comment (g4s8) [inline src/main/java/com/artipie/npm/Npm.java]: @Guseyn let's throw some exception here, logging errors may cause unpredictable behavior.

**Comment by g4s8** [comment]
@Guseyn also, don't forget to update github action files like in https://github.com/artipie/asto/blob/master/.github/workflows/maven.yml and rultor config: https://github.com/artipie/asto/blob/master/.rultor.yml

**Comment by Guseyn** [comment]
@g4s8 done.

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/artipie/npm-adapter/issues/21#issuecomment-591913490))

@g4s8 This pull request [#21](https://github.com/artipie/npm-adapter/issues/21) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/abd1c157-9a22-4921-ad22-6d72de8a1c86); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**